docs/modules/operation/pages/events/event-configuration.adoc

```diff
@@ -75,10 +75,16 @@ You can include various tokens in the description, log message, operator instruc
 These tokens will be replaced by values from the current event when the text for the event is constructed.
 Not all events will have values for all tokens, and some refer specifically to information available only in events derived from SNMP traps.
 
+Note that parameter descriptions use the percentage sign (%) as a replacement variable.
```

Comment:
The percentage sign itself is not a replacement variable. It is used as a kind of delimiter to mark a replaceable variable.